sample-apps/donet-sample-app/Controllers/AppController.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using Amazon.S3;
+using Microsoft.AspNetCore.Mvc;
+using System.Diagnostics;
+using System.Net.Http;
+using Microsoft.AspNetCore.Http.Extensions;
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Instrumentation;
+using System.Diagnostics.Metrics;
+
+namespace integration_test_app.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class AppController : ControllerBase
+    {
+        private readonly AmazonS3Client s3Client = new AmazonS3Client();
+        private readonly HttpClient httpClient = new HttpClient();
+        private readonly MetricEmitter _metricEmitter = new MetricEmitter();
+        private static Random rand = new Random(DateTime.Now.Millisecond);
+        
+
+        public AppController()
+        {
+            //Random Metrics
+            _metricEmitter.updateTotalTimeMetric(rand.Next(10,30));
+            _metricEmitter.updateTotalHeapSizeMetric(rand.Next(100,500));
+            _metricEmitter.updateTotalThreadSizeMetric(rand.Next(1,20));
+            _metricEmitter.updateCpuUsageMetric(rand.Next(1,100));
```

Comment:
Are these supposed to be configurable based to meet the sample app spec?

Reply:
I was following the design implementation of the [java-sample-app](https://github.com/aws-observability/aws-otel-community/tree/master/sample-apps/java-sample-app) and I did not see configuration options on them. Are we even testing the bounds of the metrics produced?

Reply:
What does the spec say? I believe that should be your basis for implementation.